spec/core/SpecSpec.js

```diff
@@ -357,6 +357,23 @@ describe("Spec", function() {
     }]);
   });
 
+  it("should return true when handling an exception", function() {
```

Comment:
I think the unit test is a bit nicer in this case since it is a bit easier to note which cases cause `true` or `false`.